packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md

```diff
@@ -15,7 +15,7 @@ This guide will walk you through integrating authentication into a React app wit
 6. [Connect the Routes](#connect-the-routes)
 7. [Start Your App](#start-your-app)
 
-> This guide is for `@okta/okta-signin-widget` v4.5.1 and `@okta/okta-react` v3.0.8.
+> This guide is for `@okta/okta-signin-widget` v4.5.1 and `@okta/okta-react` v4.1.0.
```

Comment:
Do we have a separate task for updating to the latest widget (now in 5.x I believe)?

Reply:
@swiftone 
I didn't see that task, I can assume, that we do not have...

Reply:
I think we should target to SIW v5.x since it's the latest version, the Jira probably is out of date. You can follow the [custom signing sample](https://github.com/okta/samples-js-react/tree/master/custom-login), which is using SIW 5.x now.

Reply:
@shuowu @swiftone 
Updated SIW to v5.1.4